xonsh/tools.py

```diff
@@ -830,14 +830,22 @@ def command_not_found(cmd):
     """Uses the debian/ubuntu command-not-found utility to suggest packages for a
     command that cannot currently be found.
     """
+    # lazy import
```

Comment:
I'm sorry for nagging you, but could you please remove this comment. It's pretty obvious what the code does.

Reply:
Oh, sure. I commented because the only other place where shutil was used it was commented like that.

Reply:
Does it? I'm twice sorry for nagging you then.